rapid7/metasploit-framework
# finish move of php / python meterpreters to metasploit-payloads

This PR updates metasploit-payloads to 1.0.10, which no longer blacklists php and python meterpreter, deletes the copies from data/meterpreter and adjusts the stages to read from the gem instead.
# Verification steps
- [x] Ensure that Python meterpreter still works, e.g. minimally run something like this:

```
./msfconsole -qx 'use exploit/multi/handler; set payload python/meterpreter/reverse_tcp; set lhost 192.168.56.1; run'
```

and connecting to port 4444 should spit out the payload
- [x] Ensure that PHP meterpreter still works, same as above.
